Code.Rmd

```diff
@@ -212,17 +453,16 @@ Some common uses of `.onLoad()` and `.onAttach()` are:
     }
     ```
     
-    Then devtools functions can use e.g. `getOption("devtools.name")` to 
-    get the name of the package author, and know that a sensible default value
-    has already been set.
+  Then devtools functions can use e.g. `getOption("devtools.name")` to get the
+  name of the package author, and know that a sensible default value has already
+  been set.
     
-*   To connect R to another programming language. For example, if you use rJava
-    to talk to a `.jar` file, you need to call `rJava::.jpackage()`. To
-    make C++ classes available as reference classes in R with Rcpp modules,
-    you call `Rcpp::loadRcppModules()`. 
-
-*   To register vignette engines with `tools::vignetteEngine()`.
+* To connect R to another programming language. For example, if you use rJava to
+  talk to a `.jar` file, you need to call `rJava::.jpackage()`. To make C++
+  classes available as reference classes in R with Rcpp modules, you call
+  `Rcpp::loadRcppModules()`. 
 
+* To register vignette engines with `tools::vignetteEngine()`.
```

Comment:
I think we can probably omit this example

Reply:
Maybe replace with a forward link handling different versions of R/dependencies?